es-models/case_centric/case_centric.mapping.yaml

```diff
@@ -265,23 +265,23 @@ properties:
   exposures:
     properties:
       alcohol_days_per_week:
-        type: long
+        type: double
       alcohol_history:
         type: keyword
       alcohol_intensity:
         type: keyword
       asbestos_exposure:
         type: keyword
       bmi:
-        type: float
```

Comment:
I'm surprised `bmi` was already a float here. I thought I remember seeing it quantized in some contexts.

The change should be good, though. I guess we'll see if we still have issues with values being quantized/truncated after we re-run mutation indexer with the new mappings.